### ISSUE TITLE
Inline edit comments on frontend.

### ISSUE THREAD

**Issue by ihorchepurnyi** [yii2mod/yii2-comments#50]
1. Add ability for admin user to edit comments on frontend.
2. Tests

**Comment by ihorchepurnyi** [comment]
Resolved by #60

**Comment by ihorchepurnyi** [comment]
```php
'modules' => [
    'comment' => [
        'class' => 'yii2mod\comments\Module',
        'enableInlineEdit' = true,
    ],
]
```
